app/config/opConfig.js

```diff
@@ -114,6 +114,12 @@ angular.module('opApp')
         // List of tags to attempt to match against WMS layer keywords.
         // If a tag cannot be found for any layer it is not added as a selection.
         // Order is important as that is the order they will be rendered in layer control.
-        recognizedTags: [ 'MODIS', 'VIIRS', 'TOPP' ]
+        recognizedTags: [ 'MODIS', 'VIIRS', 'TOPP' ],
+
+        getVersion: function(){
+          return $http({ method: 'GET', url: 'modules/version.json', timeout: 50000}).then(function (result) {
```

Comment:
Is there a reason that we're doing a version in a file that we're keeping local with the app and then HTTP getting locally?  I could understand if this version file may reside elsewhere on another server, but I doubt it will.  Also, why can't this just be a variable that we set in opConfig instead of another file that we open and do all this extra jazz with?